drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -441,6 +446,23 @@ function DragAndDropTemplates(configuration) {
         )
     };
 
+    var forwardKeyboardHelpButtonTemplate = function(ctx) {
+        return h(
+            'button.unbutton.btn-default.btn-link.'+"keyboard-help-button",
```

Comment:
This should be changed to a single literal string now that we're no longer using a variable:

```javascript
'button.unbutton.btn-default.btn-link.keyboard-help-button'
```